packages/application-shell-connectors/src/components/application-context/application-context.tsx

```diff
@@ -44,17 +44,11 @@ type TApplicationContextGroupedByResourceType = {
  *   }
  * }
  */
-// NOTE: we currently can't use Mapped Types as the babel transfomer does not
-// understand them yet: https://github.com/milesj/babel-plugin-typescript-to-proptypes/issues/23
-// type TApplicationContextDataFenceType = 'store';
-// type TApplicationContextDataFences = {
-//   // E.g. { store: {...} }
-//   [key in TApplicationContextDataFenceType]: TApplicationContextGroupedByResourceType;
-// };
-type TApplicationContextDataFences = {
-  // E.g. { store: {...} }
-  store?: TApplicationContextGroupedByResourceType;
```

Comment:
To double check: the `store` was marked as an optional property before. How does it work with a `Record`?

Reply:
🔥 that is a nice catch!
I assumed that `store` will be optional, but that is not the case. 😒
This can be rewritten to use `Partial` of `Record`, if we want.

```ts
const TDataFences = Partial<Record<DataFenceTypes, TApplicationContextGroupedByResourceType>>
```